Assets/Scripts/Score/ScoreManager.cs

```diff
@@ -0,0 +1,78 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using TMPro;
+
+public class ScoreManager : MonoBehaviour
+{
+    public static ScoreManager Instance { get; private set; }
+    public float Score
+    {
+        get => score;
+        private set { score = value; }
+    }
+
+    [SerializeField]
+    private TextMeshProUGUI scoreText;
+    [SerializeField]
+    private Transform playerTransform;
+
+    [SerializeField]
+    private float playerDistPtsMultiplier = 10.0f;
+    [SerializeField]
+    private float enemyFelledPts = 100.0f;
+    [SerializeField]
+    private float dispScoreTransSpeed = 100.0f;
+
+    private float score,
+                  displayScore,
+                  pastHighScore,
+                  startPlayerYPos;
+
+    private string sceneHighScore;
+    
+    // Pass a positive for increase, negative for decrease
+    public void ChangeScore(float amount) { Score += amount; }
```

Comment:
I think this method should be removed. Instead, the score can be added to directly.

Reply:
Removing this would lead to no public option to change the private score variable.

Reply:
Public access to changing the score isn't needed at this point, but I'm fine with keeping the method. I'm not sure if this method will be used outside of this script in the future. You could also rename `ChangeScore` to something like `AddToScore` and remove the comment.